### ISSUE TITLE
Reader | Pressing alt+enter saves empty comment

### ISSUE THREAD

**Issue by dannysellers** [department-of-veterans-affairs/caseflow#4000]
PR #3856 disabled the Save button for annotations when empty text, but the keyboard shortcut handler still functions. This triggers a 500 on the backend, which is logged to Sentry as an error (#3972).

**Comment by astewarttistatech** [comment]
**FAILED**
Still saves empty strings

**Comment by dannysellers** [comment]
This is superseded by #4015, closing.

**Comment by astewarttistatech** [comment]
blocked by #4036

**Comment by kierachell** [comment]
**PASSED**
![blank](https://user-images.githubusercontent.com/18075411/33530691-b5a8b074-d851-11e7-8542-ec56dff59cbe.gif)
